ls1intum/Artemis
# [Exam] Allow to submit early

### Checklist

### Motivation and Context
We want to provide the possibility to hand in an exam early.

### Description
- [x] I've added a button to abort the exam
- [x] I've added a back/continue button with a corresponding description to continue after the student accidentally clicked *Abort*
- [x] I've ensured that all participation related components are not shown after the student submitted

### Steps for Testing
1. Log in to Artemis
2. Participate in an exam
3. Click the *Abort* button
4. Click the *Continue* button
5. Refresh and check that you can still participate
6. Click the *Abort* button
7. Finish the exam
8. Refresh and check that you cannot participate anymore (summary page is shown)

### Screenshots
![Ohne-Titel](https://user-images.githubusercontent.com/15168372/87468217-78b79280-c619-11ea-965b-f19a6cc91950.gif)

**Responsive behavior of the exam navigation bar:**
![responsive](https://user-images.githubusercontent.com/15168372/87463391-98e35380-c611-11ea-9957-402ebd555071.gif)
